bin/console

```diff
@@ -139,6 +139,8 @@ if (array_key_exists('config-dir', $options)) {
 
 require_once dirname(__DIR__) . '/vendor/autoload.php';
 
+$_SESSION["glpi_currenttime"] = date("Y-m-d H:i:s");
+
```

Comment:
This should not be required. The session is started by the `Glpi\Kernel\Listener\PostBootListener\SessionStart` listener that is call even in CLI context.

Reply:
I agree this should not be required, but it's not properly set in password reset context; and I do not know why.
Since current PR does not fix the issue; this can be reverted and properly fixed in another PR.

Reply:
```suggestion
```